app/src/main/java/org/thoughtcrime/securesms/components/location/SignalPlace.java

```diff
@@ -0,0 +1,85 @@
+package org.thoughtcrime.securesms.components.location;
+
+import android.net.Uri;
+import android.text.TextUtils;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.android.gms.maps.model.LatLng;
+
+import org.signal.core.util.logging.Log;
+import org.thoughtcrime.securesms.maps.AddressData;
+import org.thoughtcrime.securesms.util.JsonUtils;
+
+import java.io.IOException;
+
+public class SignalPlace {
+
+  private static final String URL = "https://maps.google.com/maps";
+  private static final String TAG = Log.tag(SignalPlace.class);
+
+  @JsonProperty
+  private CharSequence name;
+
+  @JsonProperty
+  private CharSequence address;
+
+  @JsonProperty
+  private double latitude;
+
+  @JsonProperty
+  private double longitude;
+
+  public SignalPlace(@NonNull AddressData place) {
+    this.name      = "";
+    this.address   = place.getAddress();
+    this.latitude  = place.getLatitude();
+    this.longitude = place.getLongitude();
+  }
+
+  @JsonCreator
+  @SuppressWarnings("unused")
+  public SignalPlace() {}
+
+  @JsonIgnore
+  public LatLng getLatLong() {
+    return new LatLng(latitude, longitude);
+  }
+
+  @JsonIgnore
+  public String getDescription() {
+    String description = "";
+
+    if (!TextUtils.isEmpty(name)) {
+      description += (name + "\n");
+    }
+
+    if (!TextUtils.isEmpty(address)) {
+      description += (address + "\n");
+    }
+
+    description += Uri.parse(URL)
+                      .buildUpon()
+                      .appendQueryParameter("q", String.format("%s,%s", latitude, longitude))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format double: latitude, longitude. This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 67. Consider formatting this data with &#39;%f&#39; instead.